packages/timeline-state-resolver/src/integrations/sofieChef/index.ts

```diff
@@ -373,15 +410,24 @@ export class SofieChefDevice extends DeviceWithState<SofieChefState, DeviceOptio
 
 			if (!newWindow) {
 				// Removed
-				commands.push({
-					context: 'removed',
-					timelineObjId: oldWindow.urlTimelineObjId,
-					content: {
-						msgId: 0, // set later
-						type: ReceiveWSMessageType.STOP,
-						windowId: windowId,
-					},
-				})
+
+				// Is it in the mappings?
+				// We should only touch stuff that is in our mappings:
```

Comment:
Actually thinking about this a second time and now I'm even doubting this... mappings can change when route sets change and we maybe want to remove the window?

Reply:
Fair enough! I'll remove the check for now and we can revisit it later when we're explicitly looking into partial control.